docs/easypy/usages.rst

```diff
@@ -295,37 +294,98 @@ constructed and processed using python `argparse`_ module.  Please also see
                                        --clean-devices "[[device_a, device_b, device_c], [device_d, device_e]]"\
                                        --invoke-clean
 
-``--clean-image``
-    specifies images to be used for clean per device
+.. _kleenex_cli_image_format:
+
+``--clean-device-image``
+    specifies images to be used for clean per device.
 
     .. code-block:: bash
 
         bash$ pyats run job jobfile.py --testbed-file /path/to/my/testbed.yaml\
                                        --clean-file /path/to/my/clean.yaml\
                                        --invoke-clean\
-                                       --clean-image PE1:/path/to/clean_image.bin
+                                       --clean-device-image PE1:/path/to/clean_image.bin
+
+    This is equivalent to the following in YAML
+
+    .. code-block:: yaml
 
-``--clean-platform``
-    specifies images to be used for clean per OS
+        images:
+        - /path/to/clean_image.bin
+
+    To provide a list of images:
 
     .. code-block:: bash
 
         bash$ pyats run job jobfile.py --testbed-file /path/to/my/testbed.yaml\
                                        --clean-file /path/to/my/clean.yaml\
                                        --invoke-clean\
-                                       --clean-platform iosxe:/path/to/clean_image.bin
+                                       --clean-device-image PE1:/path/to/controller_image.bin\
+                                       PE1:/path/to/switch_image.bin
+
+    This is equivalent to the following in YAML:
 
-``--clean-separator``
-    character used to separator device/OS from image path within args
-    'clean-image' and 'clean-platform'
+    .. code-block:: yaml
+
+        images:
+        - /path/to/controller_image.bin
+        - /path/to/switch_image.bin
+
+    To provide images with a key structure:
 
     .. code-block:: bash
 
-        bbash$ pyats run job jobfile.py --testbed-file /path/to/my/testbed.yaml\
+        bash$ pyats run job jobfile.py --testbed-file /path/to/my/testbed.yaml\
+                                       --clean-file /path/to/my/clean.yaml\
+                                       --invoke-clean\
+                                       --clean-device-image PE1:image:/path/to/image.bin\
+                                       PE1:packages:/path/to/optional_package1\
+                                       PE1:packages:/path/to/optional_package2
+
+    This is equivalent to the following in YAML:
+
+    .. code-block:: yaml
+
+        images:
+          image:
+          - /path/to/image.bin
+          packages:
+          - /path/to/optional_package1
+          - /path/to/optional_package2
+
+    `--clean-device-image` can be used in combination with `--clean-os-image`,
+    `--clean-group-image` and `--clean-platform-image`. Conflicts are resolved
+    according to the following order: `device > group > platform > os`.
```

Comment:
good mentioning to tell. by using `.. note::`, might be better to highlight a bit more.